api/api_full.go

```diff
@@ -325,6 +325,11 @@ type FullNode interface {
 	// WalletValidateAddress validates whether a given string can be decoded as a well-formed address
 	WalletValidateAddress(context.Context, string) (address.Address, error) //perm:read
 
+	// This method is exclusively used for Non Interactive Authentication based on a SP Worker Address
+	FilIdSp(ctx context.Context, maddr address.Address, optionalArg []byte) (string, error) //perm:sign
+	// This method is exclusively used for Non Interactive Authentication based on provided Wallet Address
+	FilIdAddr(ctx context.Context, addr address.Address, optionalArg []byte) (string, error) //perm:sign
```

Comment:
* Prefix name with `Wallet` instead of `Fil` - the first word "groups" related api methods
* The comment should start with the method name, because Golang decided that that's how you should write docstrings